packages/server/modules/core/dbSchema.js

```diff
@@ -0,0 +1,40 @@
+const knex = require('@/db/knex')
+
+/**
+ * Single source of truth for DB schema in the codebase
```

Comment:
Context: https://discord.com/channels/726756379083145269/743849923803349022/954402440269140008

Basically, instead of duplicating magic strings (table, column names) everywhere and having no visibility into whether or not the thing you're referencing actually exists (is a valid table/column name), I created this single source of truth for knex instances, table names and columns names.